modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7538,6 +7538,8 @@ feature.flag.LPD-31212.description=Users can create rich text content that inclu
 feature.flag.LPD-31212.title=Disable XSS Validations in Rich Text Editor
 feature.flag.LPD-31789.description=Allow administrators to configure Google Vertex AI as text embedding provider for semantic search.
 feature.flag.LPD-31789.title=Google Vertex AI Text Embedding Provider
+feature.flag.LPD-34594.description=Users can create object hierarchies by binding objects to a root object definition. Bound objects inherit permissions and widget configuration from the root object, and their API endpoints are merged with the root object's endpoints.
+feature.flag.LPD-34594.title=Root Objects
```

Comment:
I know this was the message on the ticket, but I would rewrite it to reinforce that we must always use the "full name" of our entities. It is the way Brian prefers.
So, I would rewrite it to:
Users can create **object definition** hierarchies by binding **object definitions** to a root **object definition**. Bound **object definitions** inherit permissions and widget configuration from the root **object definition**, and their API endpoints are merged with the root **object definition's** endpoints.

I will share this feedback with Lawrence :+1:

Reply:
Done!

Reply:
Marcella's  could work but it gets bloated with the repeated "object definition" phrase. What do you think of this?
> Users can create object definition hierarchies by defining a root object definition. Child object definitions are bound to the root and inherit permissions and widget configuration from the root object definition. The API endpoints of child object definitions are merged with the root's endpoints.

I also split the second sentence because I think Brian might not like it's length and complexity.

Reply:
Thanks @lawrence-lee and @rbohl :clap: :smile:

Reply:
Thanks for the suggestion! I changed it to this.